### PR TITLE
feat: add new filed `meta_style`, collect the request information with `original` style.

### PR DESCRIPTION
**Pull request by membphis**
### What this PR does / why we need it:

feat: add new filed `meta_style`, collect the request information with `original` style.

eg:

```
send data to kafka: POST /hello?ab=cd HTTP/1.1
host: localhost
content-length: 6
connection: close

abcdef
```

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible?

### CODE REVIEW

**Review by spacewander — COMMENTED** [review]
Not doc updated?

**Comment by membphis** [comment]
will add today

File: apisix/plugins/kafka-logger.lua [hunk 1]
```diff
@@ -32,6 +32,11 @@ local buffers = {}
 local schema = {
     type = "object",
     properties = {
+        meta_style = {
```
Comment (membphis) [inline apisix/plugins/kafka-logger.lua:35]: `meta_format` maybe a better name.

any suggestion?

Comment (nic-chen) [reply apisix/plugins/kafka-logger.lua:35]: yes, `meta_format` is better

**Review by gxthrj — APPROVED** [review]

**Review by spacewander — APPROVED** [review]

**Pull request merged**